great_expectations/data_context/types/base.py

```diff
@@ -294,7 +294,7 @@ def validate_schema(self, data, **kwargs):
         elif data["config_version"] < MINIMUM_SUPPORTED_CONFIG_VERSION:
             raise ge_exceptions.UnsupportedConfigVersionError(
                 "You appear to have an invalid config version ({}).\n    The version number must be at least {}. "
-                "Please see the migration guide at https://docs.greatexpectations.io/how_to_guides/migrating_versions.html".format(
+                "Please see the migration guide at https://docs.greatexpectations.io/en/latest/how_to_guides/migrating_versions.html".format(
```

Comment:
Small note: I changed the other way because I think there's an automatic redirect to your language (I know we only have one language now, but...)

Reply:
Oh, that must be disabled then? I just changed it because the link didn’t work.